test/unit/auth_test.js

```diff
@@ -28,12 +28,14 @@ describe('building authorization entries', function () {
     );
 
     let cred = entry.credentials().address();
-    let args = cred.signatureArgs().map((v) => StellarBase.scValToNative(v));
+    let sig = cred.signature();
+
+    expect(sig.switch().name).to.be.equal('scvVec');
+    const args = StellarBase.scValToNative(sig.vec()[0]);
```

Comment:
minor nit, would this throw on `sig.vec()` to avoid  need for `expect` on type first

Reply:
good point! amended this to a length check, instead: 19fdbf7